### PR TITLE
#551 help messages and examples edited for rucio-admin CLI

### PR DESCRIPTION
**Pull request by TomasJavurek**
#551 help messages and examples edited for rucio-admin CLI

### CODE REVIEW

File: bin/rucio-admin [hunk 1]
```diff
@@ -7,7 +7,7 @@
  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
  Authors:
- - Mario Lassnig, <mario.lassnig@cern.ch>, 2012-2018
```
Comment (TomasJavurek) [inline bin/rucio-admin:10]: Marios change, should be ketp.

File: bin/rucio-admin [hunk 2]
```diff
@@ -572,7 +572,7 @@ def add_protocol_rse(args):
     proto.setdefault('extended_attributes', {})
     if args.ext_attr_json:
         proto['extended_attributes'] = args.ext_attr_json
-    if proto['scheme'] == 'srm' and not args.web_service_path:
```
Comment (TomasJavurek) [inline bin/rucio-admin:575]: Marios change. Should be ketp.

File: bin/rucio-admin [hunk 3]
```diff
@@ -624,10 +624,15 @@ def list_scopes(args):
     List scopes.
 
     """
+    scopes = []
     client = get_client(args)
-    scopes = client.list_scopes()
```
Comment (TomasJavurek) [inline bin/rucio-admin:628]: Marios change. Shall be ketp.

File: bin/rucio-admin [hunk 4]
```diff
@@ -891,10 +896,17 @@ def list_pfns(args):
     return SUCCESS
 
 
-def get_parser():
-    """
-    Returns the argparse parser.
-    """
```
Comment (TomasJavurek) [inline bin/rucio-admin:897]: Shall be ketp.

File: bin/rucio-admin [hunk 5]
```diff
@@ -1248,16 +1629,22 @@ if __name__ == '__main__':
...
+    argcomplete.autocomplete(oparser)
+
+    if len(sys.argv) == 1:
+        oparser.print_help()
+        sys.exit(FAILURE)
+
+    args = oparser.parse_args(sys.argv[1:])
+
     try:
-        if args.verbose:
```
Comment (TomasJavurek) [inline bin/rucio-admin:1252]: Marios change. Shall be kept.

File: bin/rucio-admin [hunk 6]
```diff
@@ -1248,16 +1629,22 @@ if __name__ == '__main__':
...
-            logger.setLevel(logging.DEBUG)
         start_time = time.time()
         command = commands.get(args.which)
         result = command(args)
         end_time = time.time()
         if args.verbose:
             print "Completed in %-0.4f sec." % (end_time - start_time)
         sys.exit(result)
-    except Exception as error:
-        logger.error("Strange error: {0}".format(error))
```
Comment (TomasJavurek) [inline bin/rucio-admin:1262]: Shall be ketp.

File: bin/rucio-admin [hunk 7]
```diff
@@ -920,78 +932,187 @@ def get_parser():
...
     list_account_identities_parser.set_defaults(which='list_identities')
     list_account_identities_parser.add_argument('account', action='store', help='Account name')
 
     # The set-limits command
-    set_account_limits_parser = account_subparser.add_parser('set-limits', help='Set the limits for the provided account')
+    set_account_limits_parser = account_subparser.add_parser('set-limits',
+                                                             help='Set the limits for the provided account at given RSE.',
+                                                             formatter_class=argparse.RawDescriptionHelpFormatter,
+                                                             epilog='''
+Example usge:
```
Comment (vingar) [inline bin/rucio-admin:1057]: typo. It should be "Usage example" and follow a RST format.

**Review by TomasJavurek — COMMENTED** [review]
Remove this version pls. I merged that manually with Marios changes.

**Pull request closed without merging**